samples/net/azure_iot_hub/README.rst

```diff
@@ -123,12 +146,38 @@ Optionally, follow the instructions at `Azure IoT Explorer`_ to install and conf
 #. In the `Azure IoT Explorer`_, navigate to the :guilabel:`Telemetry` tab and click :guilabel:`start`.
 #. Observe that the event messages from the device are displayed in the terminal within the specified telemetry interval.
 
+To test FOTA update:
+
+1. Navigate to the view where you can update the device twin using the `Azure IoT Explorer`_ or `Azure Portal`_.
+#. Create a ``firmware`` JSON object inside the *desired* object of the device twin document, containing information about the binary to download:
+
+   .. parsed-literal::
+      :class: highlight
+
+	{
+	    "firmware": {
+	        "fwVersion": "v0.0.2-dev",
+	        "jobId": "ca186d4b-4171-4209-a49d-700c35567d1d",
+	        "fwLocation": {
+	            "host": "my-fota-server.comt",
+	            "path": "firmware/my-app-update.bin"
+	        },
```

Comment:
**question:** can't we use a fully qualified URL by now? AFAIR the download client is now able to use that?